### ISSUE TITLE
CI not building supported coq release 8.12

### ISSUE THREAD

**Issue by Alizter** [HoTT/HoTT#1405]
We say we support coq 8.12 but the CI isn't actually checking this. We should fix this at some point. Currently, I cannot build the library in coq 8.12.2 after #1327. That was an overlay for 8.13 but we seemed to have forgotten to check 8.12 still works.

Getting the new numeral notations plugin to be backwards compatible is a separate issue.

**Comment by Alizter** [comment]
This doesn't seem to be an issue since #1413
